pyenvisalink/dsc_client.py

```diff
@@ -198,7 +198,7 @@ def handle_keypad_update(self, code, data):
             for i in range(0, 7):
                 if bits[15-i] == '1':
                     trouble_description += evl_verboseTrouble[i] + ', '
-                    if i == 2:
+                    if i == 1:
```

Comment:
`ac_present` was not being updated to `False` when `bit 1 = AC Power Lost` as per [TPI docs](http://forum.eyez-on.com/FORUM/download/file.php?id=223&sid=9e527acd838441b919657c5d636fd852), but on `bit 2 = Telephone Line Fault`. That is why many people had/have problems with `ac_present` not being set to `False` in Home Assistant when AC power is lost.

Reply:
Yo @Cinntax, if you can do the release then I'll create the update PR on HA side. Thank you!

Reply:
Thanks @tomachristian -
Sorry for the delay- version 4.1 has been deployed to pypi.